02-create.md

```diff
@@ -54,6 +54,32 @@ Desktop/    pizza.cfg
 molecules/  solar.pdf
 ~~~
 
+> ## Good names for files and directories {.callout}
+>
+> Names of files and directories can make your life an hell
```

Comment:
This sentence reads like "(All) names of files and directories can make your life an hell". How about adding a qualifier like "Complicated names of files and directories"?